integration-sqs-to-lambda/main.rs

```diff
@@ -0,0 +1,27 @@
+use aws_lambda_events::event::sqs::SqsEvent;
```

Comment:
It is more common to use `SqsEventObj` to give a structure for the message data.

Reply:
`SqsEventObj` is associated with a model, for example, if using DDB, we can do something like `SqsEventObj<aws_lambda_events::dynamodb::EventRecord>`. But then we will need a specific message structure to be sent. I wanted to make the example generic, hence used the `SqsEvent`. If can use `SqsEventObj<String>` or is there any other type of model you would suggest?